code-challenges.test.js

```diff
@@ -12,43 +12,105 @@
 
 
 // --------------------1) Create a function that takes in an array, removes the first item from the array and shuffles the remaining content.
+// use a function similar to .shift() or .pop() to remove an element within the array
+// .shift removes the first element of an array and returns that removed element
+// randomizer is Math.floor(Math.random() * array.length)
+// going to need more time as my og idea did not work.. going to work on this past monday...
 
 // a) Create a test with an expect statement using the variable provided. HINT: Check out this resource: https://jestjs.io/docs/expect#expectarraycontainingarray
-
+// describe("remRandArr", () => {
+
+//     // a test/it method, nested within the describe block, that in plain words, describes that the function does.
+//     it("takes in an array, removes the first item from the array and shuffles the remaining content", () => {
+  
+//       //an expect method, nested within the test block, calling on the hello() function, followed by the .toEqual() matcher that checks the expected output of the function return.
+//       expect(remRandArr(colors1)).toEqual(["yellow", "blue", "pink", "green"])
+//       expect(remRandArr(colors2)).toEqual(["saffron", "aquamarine", "periwinkle", "indigo", "ochre"])
+//     })
+//   })
 var colors1 = ["purple", "blue", "green", "yellow", "pink"]
-// Expected output example (can be a different order): ["yellow", "blue", "pink", "green"]
+// // Expected output example (can be a different order): ["yellow", "blue", "pink", "green"]
```

Comment:
Make sure you uncomment the code you are submitting so I can track your process.